portal/header.ms

```diff
@@ -34,10 +37,6 @@
 </head>
 <body class="{{#connected}}logged{{/connected}}">
 
-  <h1 id="logo" class="logo">
-      <img src="assets/img/logo-ynh-white.svg"/><span class="element-invisible">Yunohost</span>
-  </h1>
```

Comment:
Why remove this tho ? D:

Reply:
Hey, yes we discussed a bit about this in #121 with @lfuelling.
I had chosen to remove the logo here because it was confusing to have a logo that's clickable in the panel and not clickable in the homepage.

In the end, IMHO the best option would be to have a logo that's always there and a button that's clickable, so yes, maybe not remove those lines. And also the styles necessary for it.

Reply:
#121 reverts this change and also fixes another issue with the overlay (see my review in #117).

My approach to the logo would be to disable it in a theme when it's not wanted, but maybe this also could be done using configuration (a bit overkill imo).